libs/judicial-system/formatters/src/lib/formatters.spec.ts

```diff
@@ -147,6 +148,62 @@ describe('formatRestrictions', () => {
   })
 })
 
+describe('formatAlternativeTravelBanRestrictions', () => {
+  test('should return formatted restrictions for no restrictions', () => {
+    // Arrange
+    const accusedGender = CaseGender.MALE
+    const custodyRestrictions: Array<CaseCustodyRestrictions> = []
+
+    // Act
+    const res = formatAlternativeTravelBanRestrictions(
+      accusedGender,
+      custodyRestrictions,
+    )
+
+    // Assert
+    expect(res).toBe('Sækjandi tekur fram að farbannið sé án takmarkana.')
+  })
+
+  test('should return formatted restrictions for one restriction', () => {
+    // Arrange
+    const accusedGender = CaseGender.FEMALE
+    const custodyRestrictions = [
+      CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_CONFISCATE_PASSPORT,
+    ]
+
+    // Act
+    const res = formatAlternativeTravelBanRestrictions(
+      accusedGender,
+      custodyRestrictions,
+    )
+
+    // Assert
+    expect(res).toBe(
+      'Sækjandi tekur fram að farbannið verði með takmörkunum. Að kærðu verði gert að afhenda vegabréfið sitt.',
+    )
+  })
+
+  test('should return formatted restrictions for all restrictions', () => {
+    // Arrange
+    const accusedGender = CaseGender.OTHER
+    const custodyRestrictions = [
+      CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_CONFISCATE_PASSPORT,
+      CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_REQUIRE_NOTIFICATION,
+    ]
+
+    // Act
+    const res = formatAlternativeTravelBanRestrictions(
+      accusedGender,
+      custodyRestrictions,
+    )
+
+    // Assert
+    expect(res).toBe(
+      'Sækjandi tekur fram að farbannið verði með takmörkunum. Að kærða verði gert að tilkynna sig. Að kærða verði gert að afhenda vegabréfið sitt.',
```

Comment:
I think this wording is weird. Something like "Sækjandi tekur fram að farbannið verði með takmörkunum, að kærða verði gert að tilkynna sig og að kærða verði gert að afhenda vegabréfið sitt." sounds more natural to me. Just a thought, not a request for change.